### PR TITLE
Automate switch to using brew-installed bash

### PR DESCRIPTION
**Pull request by jeffbyrnes**
As requested on #656.

### CODE REVIEW

**Comment by jeffbyrnes** [comment]
@kingscooty, here you are!

**Comment by KingScooty** [comment]
Nice! 

@mathiasbynens this should take care of properly setting up bash4. 

- `bash --version`
- `echo $BASH_VERSION`

should both report `v4 `.

I think that means `~/bin/bash` can be deprecated??

File: brew.sh [hunk 1]
```diff
@@ -26,6 +26,12 @@ brew install bash
 brew tap homebrew/versions
 brew install bash-completion2
 
+# Switch to using brew-installed bash as default shell
+if ! fgrep -q '/usr/local/bin/bash' /etc/shells; then
+  echo '/usr/local/bin/bash' >> /etc/shells;
```
Comment (mathiasbynens) [inline brew.sh]: This needs `sudo` and thus `tee`.

Comment (KingScooty) [inline brew.sh]: It was actually changed due to this comment - https://github.com/mathiasbynens/dotfiles/issues/656#issuecomment-216432628

Comment (mathiasbynens) [inline brew.sh]: That comment seems wrong to me. Did you test this?

Comment (KingScooty) [inline brew.sh]: Just tested this now, you're right, it needs `sudo` and `tee`.

Comment (jeffbyrnes) [inline brew.sh]: I knew there was a reason for `tee`, but I blindly followed the suggestion on the other PR. That'll teach me: trust, but verify. 

I'm AFK for a couple of days, I'll get this back to its previous incarnation & push that up to get this working again.

**Comment by jeffbyrnes** [comment]
@mathiasbynens I’ve finally gotten back to this & updated it.

**Comment by mathiasbynens** [comment]
Thanks, everyone!

**Pull request closed without merging**